__docs__/wonder-blocks-button/button.stories.tsx

```diff
@@ -120,51 +122,49 @@ export const styles: StyleDeclaration = StyleSheet.create({
     },
 });
 
-export const Variants: StoryComponentType = {
-    render: () => (
-        <View>
-            <View style={{flexDirection: "row"}}>
-                <Button onClick={() => {}}>Hello, world!</Button>
-                <Strut size={16} />
-                <Button onClick={() => {}} kind="secondary">
-                    Hello, world!
-                </Button>
-                <Strut size={16} />
-                <Button onClick={() => {}} kind="tertiary">
-                    Hello, world!
-                </Button>
-            </View>
+export const Variants: StoryComponentType = () => (
```

Comment:
**note:** Removed the `render` fn to be able to reuse the story in the `KhanmigoTheme` story at the end of this file. This change caused a lot of indentation changes.